apps/src/templates/sectionAssessments/FreeResponseBySurveyQuestionContainer.jsx

```diff
@@ -24,7 +34,11 @@ class FreeResponseBySurveyQuestionContainer extends Component {
         <h2>{i18n.studentFreeResponseAnswers()}</h2>
         {freeResponsesByQuestion.map((question, index) => (
           <div key={index}>
-            <h3>{`${question.questionNumber}. ${question.questionText}`}</h3>
+            <div style={styles.text}>
+              <span>{`${question.questionNumber}. ${question.questionText}`.slice(0, QUESTION_CHARACTER_LIMIT)}</span>
```

Comment:
I think it's cleaner to write this like this because we care mostly about slicing the question text:
```
{`${question.questionNumber}. ${question.questionText.slice(0, QUESTION_CHARACTER_LIMIT)}`}
```

Reply:
Updated!